templates/download/iot/intel-iot.html

```diff
@@ -93,6 +93,25 @@ <h2 class="p-heading--3">Backed by Intel's partner ecosystem</h2>
   </div>
 </section>
 
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Real-time Ubuntu Beta optimised for Intel SoCs</h2>
+      <p>Optimised <a href="/blog/real-time-ubuntu-is-now-generally-available">real-time Ubuntu</a> is now available on 12th Gen Intel&reg; Core&trade; processors in Beta.</p>
+      <p>Coupled with Intel&reg; Time Coordinated Computing (Intel&reg; TCC) and Time Sensitive Networking (IEEE TSN), the real-time Ubuntu kernel delivers industrial-grade performance to the time-bound workloads of industrial enterprises.</p>
+      <p><a href="https://register.gotowebinar.com/register/8712572348523284318?source=Canonical">Watch the joint webinar by Canonical and Intel on how real-time Ubuntu on Intel accelerates industrial transformation</a></p>
+      <p>Real-time Ubuntu on Intel SoCs is available via <a href="/pro">Ubuntu Pro</a>, Canonical's enterprise security and compliance subscription, covering all aspects of open infrastructure. With an <a href="/pro/dashboard">Ubuntu Pro subscription</a>, launching the kernel is as easy as:</p>
+      <div class="p-code-snippet">
+        <pre class="p-code-snippet__block--icon"><code>pro attach &lt;token&gt;</code></pre>
+        <pre class="p-code-snippet__block--icon"><code>pro enable realtime-kernel --access-only</code></pre>
+        <pre class="p-code-snippet__block--icon"><code>apt install ubuntu-intel-iot-realtime</code></pre>
+      </div>
+      <p>Are you interested in running real-time Ubuntu on Intel SoCs?</p>
+      <p><a href="/kernel/real-time/contact-us" class="p-button--positive">Get in touch</a></p>
```

Comment:
```suggestion
      <p><a href="/kernel/real-time/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a></p>
```

Reply:
I don't want to do that because that posts the data to a different formID